cmd/zbClient.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2021 Matheus Cruz <matheuscruz.dev@gmail.com>
```

Comment:
@mcruzdev we don't use these headers with names like this. We leave the License only here.